pragma/publisher/client.py

```diff
@@ -79,3 +87,174 @@ def fetch_sync(self) -> List[any]:
             data = fetcher.fetch_sync()
             results.extend(data)
         return results
+
+
+class PragmaAPIClient(PragmaClient):
+
+    api_base_url=os.getenv("PRAGMA_API_BASE_URL"),
+    api_key=os.getenv("PRAGMA_API_KEY"),
+    
+    
+    @staticmethod
+    def convert_to_publisher(client: PragmaClient):
+        client.__class__ = PragmaAPIClient
+        return client
+
+
+    def api_get_ohlc(
```

Comment:
can we make methods async by default and use the decorator to create sync methods ? cc #65

Reply:
Yep, a good point, I will do a general implementation in another pr